static/app/components/replays/replayFullscreenButton.tsx

```diff
@@ -0,0 +1,22 @@
+import {Button} from 'sentry/components/button';
+import {IconContract, IconExpand} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import useIsFullscreen from 'sentry/utils/window/useIsFullscreen';
+
+type ReplayFullscreenButtonProps = {
```

Comment:
`Props` is probably sufficient, it's a small file, no need to be wordy

Reply:
That's fair, I have a vs code expander that auto names the props this way but I'll name it Props to match the rest of the replays components